test/integration/targets/known_hosts/aliases

```diff
@@ -0,0 +1,2 @@
+needs/root
```

Comment:
You can run the test locally as a non-root user to see if it passes (make sure you remove the `needs/root` alias first):

`test/runner/ansible-test integration known_hosts -v`